asm/bzero.s

```diff
@@ -9,76 +9,65 @@
 
 .balign 16
 
-glabel bzero
-/* 005050 80004450 28A1000C */  slti  $at, $a1, 0xc
-/* 005054 80004454 1420001D */  bnez  $at, .bytezero
-/* 005058 80004458 00041823 */   negu  $v1, $a0
-/* 00505C 8000445C 30630003 */  andi  $v1, $v1, 3
-/* 005060 80004460 10600003 */  beqz  $v1, .blkzero
-/* 005064 80004464 00A32823 */   subu  $a1, $a1, $v1
-/* 005068 80004468 A8800000 */  swl   $zero, ($a0)
-/* 00506C 8000446C 00832021 */  addu  $a0, $a0, $v1
+LEAF(bzero)
+    slti    $at, $a1, 0xC
+    bnez    $at, .bytezero
+     negu   $v1, $a0
+    andi    $v1, $v1, 3
+    beqz    $v1, .blkzero
+     subu   $a1, $a1, $v1
+    swl     $zero, ($a0)
+    addu    $a0, $a0, $v1
 .blkzero:
```

Comment:
not your change but
```suggestion
.blockzero:
```
it took me a very long time to figure out what this blk meant